### ISSUE TITLE
Text Structure and Color Contrast

### ISSUE THREAD

**Issue by nonsponsored** [nonsponsored/BeBestNDS#7]
### Site

trumpcard.gov

### Page URL

https://trumpcard.gov

### Type of Issue

Accessibility

### Description of Issue

1. There are multiple `h1` tags on the page.
2. Active and inactive Special Benefits button text colors fail color contrast.
3. Color contrast fails on the description copy of Special Benefits content.
4. Description copy of FAQs fail color contrast.

### Device Type (Optional)

None

### Operating System (Optional)

None

### Browser (Optional)

None

### Confirmation

- [x] I confirm this is a factual report of a technical, usability, or accessibility issue.
